cocoa/src/toga_cocoa/widgets/canvas.py

```diff
@@ -269,25 +269,28 @@ def _render_string(self, text, font, **kwargs):
 
     # Although the native API can measure and draw multi-line strings, this makes the
     # line spacing depend on the scale factor, which messes up the tests.
-    def _line_height(self, font):
-        # descender is a negative number.
-        return ceil(font.native.ascender - font.native.descender)
+    def _line_height(self, font, line_height):
+        if line_height is None:
+            # descender is a negative number.
+            return ceil(font.native.ascender - font.native.descender)
+        else:
+            return font.native.ascender * line_height
```

Comment:
Are you sure this should be `ascender`, and not `fontSize`? The two are likely very similar, but I'm not sure they're *required* to be identical.